sessions/sessions.go

```diff
@@ -3138,20 +3183,30 @@ func (sS *SessionS) BiRPCv1ProcessEvent(clnt rpcclient.RpcClientConnection,
 	if argsFlagsWithParams.HasKey(utils.MetaThresholds) {
 		tIDs, err := sS.processThreshold(args.CGREvent, args.ArgDispatcher,
 			argsFlagsWithParams.ParamsSlice(utils.MetaThresholds))
-		if err != nil {
-			return err
+		if err != nil && err.Error() != utils.ErrNotFound.Error() {
+			utils.Logger.Warning(
+				fmt.Sprintf("<%s> error: %s processing event %+v with ThresholdS.",
+					utils.SessionS, err.Error(), args.CGREvent))
+			withErrors = true
 		}
 		rply.ThresholdIDs = &tIDs
 	}
 	// process stats if required
 	if argsFlagsWithParams.HasKey(utils.MetaStats) {
 		sIDs, err := sS.processStats(args.CGREvent, args.ArgDispatcher,
 			argsFlagsWithParams.ParamsSlice(utils.MetaStats))
-		if err != nil {
-			return err
+		if err != nil &&
+			err.Error() != utils.ErrNotFound.Error() {
+			utils.Logger.Warning(
+				fmt.Sprintf("<%s> error: %s processing event %+v with StatS.",
+					utils.SessionS, err.Error(), args.CGREvent))
+			withErrors = true
 		}
 		rply.StatQueueIDs = &sIDs
 	}
+	if withErrors {
```

Comment:
maybe simplify it with : if withErrors { err=xxx } return

So we can have only one return, looks fancier ;)